app/src/main/java/com/hedvig/app/util/featureflags/FeatureRuntimeBehavior.kt

```diff
@@ -0,0 +1,32 @@
+package com.hedvig.app.util.featureflags
+
+import androidx.annotation.VisibleForTesting
+import com.hedvig.app.feature.settings.MarketManager
+import java.util.concurrent.CopyOnWriteArrayList
+
+class FeatureRuntimeBehavior(
+    val marketManager: MarketManager,
+    isDebugBuild: Boolean
+) {
+
+    @VisibleForTesting
+    internal val providers = CopyOnWriteArrayList<FeatureFlagProvider>()
+
+    init {
+        if (isDebugBuild) {
```

Comment:
It would be super-cool to provide these via multi-binding and have them reside only in the appropriate source sets, but Koin doesn't really support multi-binding 😞 I think we can skip it for now, but it would be really cool!